e2e-tests/tailwind.config.ts

```diff
@@ -0,0 +1,15 @@
+import type { StudioTailwindConfig } from "@yext/studio";
+
+export default {
+  content: ["./src/**/*.{ts,tsx}"],
+  theme: {
+    extend: {
+      colors: {
+        primary: "#aa00ff",
+      },
+      fontSize: {
+        medium: "14px",
```

Comment:
Do we actually use any of these theme extensions from this config in the tests?

Also, I thought the way we had set up Tailwind currently, is that each test would need to specify the path to the Tailwind config for it to actually be copied over to the cloned playground for that test?

Reply:
I see-- yeah, It looks like we don't use the extended theme.  I also saw that we didn't use it in the test-site either, so I removed it here as well.

And yes!  Thanks for the catch, I see what you're saying.  I added the tailwind to `studioTest` so that the tailwind is correctly copied over into the playground.  I tested with a custom theme extension, and it works!